playbooks/appsemblerPlaybooks/ginkgo_basic.yml

```diff
@@ -85,3 +85,5 @@
     - role: ecommerce_theme
       when: ENABLE_ECOMMERCE
       tags: ['ecommerce_theme_role']
+    - role: discovery
+      when: SANDBOX_ENABLE_DISCOVERY
```

Comment:
Is there any reason we don't want to enable discovery on all customer deployments by default? 
Isn't it needed for the course catalog? 

Also, where did the `SANDBOX_` part of the variable come from? That seems unnecessary.